snuba/clusters/cluster.py

```diff
@@ -44,8 +45,12 @@ class ClickhouseClientSettings(Enum):
         {"load_balancing": "in_order", "replication_alter_partitions_sync": 2}, 10000
     )
     OPTIMIZE = ClickhouseClientSettingsType({}, 10000)
-    QUERY = ClickhouseClientSettingsType({"readonly": 1}, None)
-    TRACING = ClickhouseClientSettingsType({"readonly": 2}, None)
+    QUERY = ClickhouseClientSettingsType(
+        {"readonly": 1}, None, settings.CLICKHOUSE_READONLY_USER
+    )
```

Comment:
We use the QUERY settings for all of our queries in the Snuba API and subscriptions. We cannot use the readonly user (which has much lower limits for that)

Reply:
got it, I'll add another setting